pkgs/development/python-modules/cddlparser/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  # dependencies
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cddlparser";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-EcGxDRGFri0nS1GpoPXnijE7xBg2RuBle7U8Byiw2ns=";
+  };
```

Comment:
Can you fetch the source from GitHub? This is now preferred against using pypi (upstream fetched from source, can also contain tests)